internal/detector/application_lister.go

```diff
@@ -136,9 +136,10 @@ func isDirectory(path string) (bool, error) {
 	return fileInfo.IsDir(), err
 }
 
-func expandJarPaths(cwd string, paths []string) ([]string, error) {
+func expandJarPaths(cwd string, paths []string, verbose bool) ([]string, error) {
```

Comment:
The verbose param does not seem to be used here.
Then if you remove it here, and you  get back to the caller, it won't be used anymore too.

In the end once you setup the log level you don't need this info, except for tests.
Is it worth it?